amplipi/eeprom.py

```diff
@@ -3,58 +3,122 @@
 These are used to identify features and device types among other things."""
 
 from dataclasses import dataclass
-from enum import Enum
+from enum import IntEnum
+from struct import pack, unpack, calcsize
 from time import sleep
-from typing import List, Tuple
-import smbus2 as smbus
+from typing import List, Optional, Union, Tuple
```

Comment:
Is your setup showing you unused imports?
```suggestion
from typing import List, Optional, Union
```